src/Core/src/Response.php

```diff
@@ -157,6 +158,13 @@ public function getStatusCode(): int
 
     public function toStream(): ResultStream
     {
+        if (\method_exists($this->httpResponse, 'toStream')) {
```

Comment:
nop, but it's not part of the interface: https://github.com/symfony/symfony/issues/34834

In 5.1, we the streamWrapper will be able to create the stream for us

Did you get Why I did that?
The `client->stream()` store is not rewindable. BUT it store the content in a private `php://temp` resource.
Here, I create a `ResultStream` using this private resource instead of the `$client->stream` callback